trulens_eval/trulens_eval/feedback/provider/openai.py

```diff
@@ -3,12 +3,14 @@
 from typing import Dict, Mapping, Optional, Sequence
 
 import openai
+from openai import OpenAI
+
+client = OpenAI()
```

Comment:
Is there a reason we do this here? What if the user has multiple API keys they want to use for different apps?

Reply:
May be a little more involved to make changes, but it may be better if something (not sure what's the best class to implement it among OpenAI, OpenAI endpoint, etc.) accepted the OpenAI client instance.

Reply:
I think we can also remove this.